<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="4" total="5" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-06-17T14:07:22 IST" name="Default suite" finished-at="2023-06-17T14:07:43 IST" duration-ms="20961">
    <groups>
    </groups>
    <test started-at="2023-06-17T14:07:22 IST" name="Default test" finished-at="2023-06-17T14:07:43 IST" duration-ms="20961">
      <class name="com.qa.testcases.EditCustomerValidations">
        <test-method is-config="true" signature="startTesting()[pri:0, instance:com.qa.testcases.EditCustomerValidations@294425a7]" started-at="2023-06-17T14:07:22 IST" name="startTesting" finished-at="2023-06-17T14:07:38 IST" duration-ms="16371" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startTesting -->
        <test-method signature="verifyWithValidCustomerId()[pri:5, instance:com.qa.testcases.EditCustomerValidations@294425a7]" started-at="2023-06-17T14:07:38 IST" name="verifyWithValidCustomerId" finished-at="2023-06-17T14:07:42 IST" duration-ms="4433" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Edit Customer] but found [Connection failed: Access denied for user 'root'@'localhost' (using password: NO)]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Edit Customer] but found [Connection failed: Access denied for user 'root'@'localhost' (using password: NO)]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.testcases.EditCustomerValidations.verifyWithValidCustomerId(EditCustomerValidations.java:123)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyWithValidCustomerId -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.testcases.EditCustomerValidations@294425a7]" started-at="2023-06-17T14:07:42 IST" name="tearDown" finished-at="2023-06-17T14:07:43 IST" duration-ms="130" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.testcases.EditCustomerValidations -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
